docroot/modules/custom/sitenow_articles/sitenow_articles.install

```diff
@@ -16,3 +16,21 @@ function sitenow_articles_update_8001() {
     ->set('display.page_articles.display_options.display_extenders.metatag_display_extender.metatags.description', 'The latest articles from [site:name].')
     ->save();
 }
+
+/**
+ * Add numeric validation to articles view arguments.
+ */
+function sitenow_articles_update_9001() {
```

Comment:
Is this the right numbering?

Reply:
We've been doing it wrong:
```
Naming and documenting your function
For each change in a module that requires one or more actions to be performed when updating a site, add a new implementation of hook_update_N() to your mymodule.install file (assuming mymodule is the machine name of your module). Implementations of hook_update_N() are named (module name)_update_(number). The numbers are normally composed of three parts:

1 or 2 digits for Drupal core compatibility (Drupal 8, 9, 10, etc.). This convention must be followed.
1 digit for your module's major release version; for example, for 8.x-1.* use 1, for 8.x-2.* use 2, for Core 8.0.x use 0, and for Core 8.1.x use 1. This convention is optional but suggested for clarity.
2 digits for sequential counting, starting with 01. Note that the x000 number can never be used: the lowest update number that will be recognized and run for major version x is x001.
Examples:

node_update_8001(): The first update for the Drupal 8.0.x version of the Drupal Core node module.
mymodule_update_8101(): The first update for your custom or contributed module's 8.x-1.x versions.
mymodule_update_8201(): The first update for the 8.x-2.x versions.
```
https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21Extension%21module.api.php/function/hook_update_N/9.1.x

Reply:
9.1.3 so 9101?

Reply:
> 1 digit for your module's major release version

We don't technically have a release for this module so aren't we on 9001 still? Although we do write out the release version on deployment so maybe 9301 for 3.4.*?

Naming things is hard.

Reply:
Lets save this for another day. :)